### PR TITLE
mint 0.7.1 (new formula)

### PR DESCRIPTION
**Pull request by Lutzifer**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mint.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Mint < Formula
+  desc "Dependency manager that installs and runs Swift command-line tool packages"
+  homepage "https://github.com/yonaskolb/Mint"
+  url "https://github.com/yonaskolb/Mint/archive/0.7.1.tar.gz"
+  sha256 "986dd967b818ee044ed0568135db8be2083ad6c3995fe6281ada9b42b224a2ec"
+  head "https://github.com/yonaskolb/Mint.git"
+
+  depends_on :xcode
```
Comment (fxcoudert) [inline Formula/mint.rb]: Looks to me like this requires Xcode 9 (or later), but only at build time. If so, the correct idiom is:

`depends_on :xcode => ["9.0", :build]`

File: Formula/mint.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Mint < Formula
+  desc "Dependency manager that installs and runs Swift command-line tool packages"
+  homepage "https://github.com/yonaskolb/Mint"
+  url "https://github.com/yonaskolb/Mint/archive/0.7.1.tar.gz"
+  sha256 "986dd967b818ee044ed0568135db8be2083ad6c3995fe6281ada9b42b224a2ec"
+  head "https://github.com/yonaskolb/Mint.git"
```
Comment (fxcoudert) [inline Formula/mint.rb]: No head for new formulas

File: Formula/mint.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on :xcode
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/mint"
```
Comment (fxcoudert) [inline Formula/mint.rb]: We need a test that exercises the functionality of the software.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by Lutzifer** [comment]
@fxcoudert Thanks for the fast feedback, I hope it now is as requested.

**Review by fxcoudert — DISMISSED** [review]

**Comment by fxcoudert** [comment]
Thanks @Lutzifer for your contribution to Homebrew!

**Comment by Lutzifer** [comment]
We can revisit "testing by building" when https://github.com/yonaskolb/Mint/issues/46 is implemented.

**Review by fxcoudert — APPROVED** [review]

**Comment by Lutzifer** [comment]
@fxcoudert thx for the support!

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Pull request closed without merging**